src/celeritas/ext/detail/GeantBremsstrahlungProcess.cc

```diff
@@ -128,7 +128,6 @@ void GeantBremsstrahlungProcess::InitialiseEnergyLossProcess(
             em_model->SetLowEnergyLimit(energy_limit);
             em_model->SetHighEnergyLimit(energy_max);
             em_model->SetSecondaryThreshold(em_parameters->BremsstrahlungTh());
-            em_model->SetLPMFlag(em_parameters->LPM());
```

Comment:
This line may be required with older Geant4 versions to set the LPM flag correctly, since the relativistic model doesn't get it from the EM parameters in that case.

Reply:
Okay, you are right. I will put a version guard here.  Thanks.